answerking_app/tests/test_categories.py

```diff
@@ -253,92 +244,37 @@ def test_post_invalid_details_returns_bad_request(self):
         self.assertEqual(expected_failure_error, actual)
         self.assertEqual(response.status_code, 400)
 
-    def test_put_valid_with_items_returns_ok(self):
```

Comment:
got rid of this test as it was testing whether you can edit a name and add an Item simultaneously which I don't think is necessary and isn't how the requests are built currently. It would work if you got the existing items and added the new item to it. but that is just a simple update which already has a test.